includes/compat/class-wc-stripe-email-failed-refund.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Base class for the email sent to the admin and customers when a refund fails.
+ *
+ * @since 9.6.0
+ */
+abstract class WC_Stripe_Email_Failed_Refund extends WC_Email_Failed_Order {
+	/**
+	 * Returns the list of template parameters.
+	 *
+	 * @return array
+	 */
+	abstract public function get_template_params();
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		$this->title   = __( 'Refund failed', 'woocommerce-gateway-stripe' );
```

Comment:
After seeing this in context, I think we should be more specific with the title so it's clearer that it's a Stripe refund.
```suggestion
		$this->title   = __( 'Stripe refund failed', 'woocommerce-gateway-stripe' );
```

![woo_email_settings](https://github.com/user-attachments/assets/49c80436-77ae-4bf6-9b1b-fd88b8c2a293)